components/infobox/commons/infobox_building.lua

```diff
@@ -36,6 +36,7 @@ end
 ---@return string
 function Building:createInfobox()
 	local args = self.args
+	local Customizable = CustomizableFactory.createCustomizable(self.injector)
```

Comment:
Would it make sense/be possible to move this into Infobox basic instead of repeating it in any specialization?